src/spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -74,10 +87,17 @@ def get_traces(
         dtype = traces.dtype
 
         traces_fft = np.fft.fft(traces, axis=0)
-        gauss_low = self._create_gaussian(traces.shape[0], self.freq_min)
-        gauss_high = self._create_gaussian(traces.shape[0], self.freq_max)
 
-        filtered_fft = traces_fft * (gauss_high - gauss_low)[:, None]
+        if self.freq_max is not None:
+            pos_factor = self._create_gaussian(traces.shape[0], self.freq_max)
+        else:
+            pos_factor = np.ones((traces.shape[0],))
```

Comment:
Shouldn't that be `np.zeros`?

Reply:
With np.zeros the resulting traces are negative when applying highpass. 
If the idea for high pass is to remove the filtered from the original signal then I think we want (1 - filtered) rather than (0 - filtered), no?

Reply:
Ah yes you're right my bad,
I guess I shouldn't be reviewing PRs late in the day ^^'